managed/CounterStrikeSharp.API/Modules/Memory/VirtualFunctions.cs

```diff
@@ -10,9 +10,29 @@ namespace CounterStrikeSharp.API.Modules.Memory;
 
 public static class VirtualFunctions
 {
+    public enum AcquireMethod : int
+    {
+        PickUp = 0,
+        Buy,
+    };
+
+    public enum AcquireResult : int
+    {
+        Allowed = 0,
+        InvalidItem,
+        AlreadyOwned,
+        AlreadyPurchased,
+        ReachedGrenadeTypeLimit,
+        ReachedGrenadeTotalLimit,
+        NotAllowedByTeam,
+        NotAllowedByMap,
+        NotAllowedByMode,
+        NotAllowedForPurchase,
+        NotAllowedByProhibition,
+    };
+
```

Comment:
this should not be in here. maybe consider `Modules/Utils/*`, create files named the same as the enum

```
Models/Utils/AcquireMethod.cs
Models/Utils/AcquireResult.cs
```